frontend/src/i18n/locales/de.yaml

```diff
@@ -69,6 +69,7 @@ user:
   settings: Nutzereinstellungen
   your-page: Persönliche Seite
   manage-content: Inhalte verwalten
+  logged-in-as: Angemeldet als
```

Comment:
Mh good point. However, that would also mean the design has to change, because that string is shown differently than the username. So I thiiink I would actually keep it like that for now?